gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/OverflowOplog.java

```diff
@@ -1409,9 +1407,16 @@ public void clear() {
       this.value = null;
     }
 
-    private void write(ByteBuffer buffer, final int byteLength) throws IOException {
-      final int position = buffer.position();
+    private void write(ByteBuffer buffer) throws IOException {
       ByteBuffer bb = getOLF().writeBuf;
+      if (!this.value.buffer.writeSerializationHeader(buffer, bb)) {
+        flush();
+        if (!this.value.buffer.writeSerializationHeader(buffer, bb)) {
```

Comment:
Where this serialization header is getting used to avoid extra copy at the channel level. Is it so that all GfxdSerializable do have a common handling somewhere?